Formula/ffmpeg.rb

```diff
@@ -77,10 +77,18 @@ class Ffmpeg < Formula
   depends_on "libbs2b" => :optional
   depends_on "rubberband" => :optional
   depends_on "zimg" => :optional
-  depends_on "openh264" => :optional
   depends_on "xz" => :optional
   depends_on "libebur128" => :optional
 
+  depends_on "nasm" if build.with? "openh264"
```

Comment:
Yup you're correct https://github.com/Homebrew/homebrew-core/blob/master/Formula/openh264.rb#L15